kb/temporal-platform-limits-sheet.md

```diff
@@ -43,3 +43,15 @@ Hard limits fail with an error; soft limits produce a warning log on the server
     - `limit.numPendingCancelRequests.error`
     - `limit.numPendingChildExecutions.error`
 - [Search Attributes maximums](/visibility/#search-attributes-maximums)
+
+**Default limits for Temporal Cloud**
+
+- **Account level**
+  - Namespaces: 10
+  - Prometheus endpoint Retention Period: 1 month
+- **Namespace level**
+  - Actions per second: 200
```

Comment:
with possibility to spike up to 2x

Reply:
Can we update that actions per second is peak

Reply:
@rohitgup14: Which of the following would be mo'better?
- Peak actions per second: 200 (can spike to 400)
- Actions per second (peak): 200 (can spike to 400)
- Some other variation